src/controllers/userController.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-else-return */
+/* eslint-disable keyword-spacing */
+/* eslint-disable no-unused-vars */
+/* eslint-disable require-jsdoc */
+import sgMail from '@sendgrid/mail';
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcrypt';
+import Util from '../utils/Utils';
+import Mail from '../utils/Mail';
+import userService from '../services/UserServices';
+import AuthenticationHelper from '../utils/AuthHelper';
+
+const util = new Util();
+const mail = new Mail();
+const { jwtSignReset, jwtVerify } = AuthenticationHelper;
+class UserController {
+  static async reset(req, res) {
+    // check whether email address is recorded
+    const searchUser = await userService.findUserByEmail(req.body.email);
+    if (!searchUser) {
+      util.setError(409, 'Email not found');
```

Comment:
I think the status code should be a `404` because the email is not found.

Reply:
Let me change it right away